echoshader/new_version/get_box.py

```diff
@@ -1,23 +1,79 @@
 import holoviews
+import numpy
 
 
-# get box stream data from box select
-def get_box_stream(echogram):
+def get_box_stream(source_pic, bounds: tuple = None):
+    """
+    Get the box stream data from the box select.
+
+    Args:
+        source_pic: The source picture or plot in holoviews ecosystem.
+        bounds (tuple, optional): The bounds of the box select.
+
+    Returns:
+        holoviews.streams.BoundsXY: The box stream data.
+    """
     box_stream = holoviews.streams.BoundsXY(
-        source=echogram,
+        source=source_pic,
         bounds=(
-            echogram.lbrt[0],
-            echogram.lbrt[1],
-            echogram.lbrt[2],
-            echogram.lbrt[3],
-        ),
+            source_pic.lbrt[0],
+            source_pic.lbrt[1],
+            source_pic.lbrt[2],
+            source_pic.lbrt[3],
+        )
+        if bounds is None
+        else bounds,
     )
 
     return box_stream
 
 
-# plot box based on box select
+def get_lasso_stream(source_pic, geometry: numpy.array = None):
+    """
+    Get the lasso stream data from the box select.
+
+    Args:
+        source_pic: The source picture or plot in holoviews ecosystem.
+        geometry (numpy.array, optional): The geometry coordinates of the lasso.
+
+    Returns:
+        holoviews.streams.Lasso: The lasso stream data.
+    """
+    lasso_stream = holoviews.streams.Lasso(
+        source=source_pic,
+        geometry=numpy.array(
+            [
+                [
+                    source_pic.lbrt[0],
```

Comment:
What does "lbrt" stand for?

Reply:
ok it is the corners -- how about changing it to `.bbox`?

Reply:
Hey @leewujung: This is a built-in property in holoviews, we can not change it